src/main/java/com/mavericsystems/authenticationauthorizationservice/controller/UserController.java

```diff
@@ -45,25 +48,34 @@
     @PostMapping("/login")
     public ResponseEntity<JWTResponse> login(@RequestBody LoginRequest loginRequest) throws Exception {
         try {
-             authenticationManager.authenticate(new UsernamePasswordAuthenticationToken(loginRequest.getEmail(), loginRequest.getPassword()));
-        } catch (BadCredentialsException ex) {
-            throw new Exception("Invalid credentials", ex);
+            final UserDetails userDetails = userService.loadUserByUsername(loginRequest.getEmail());
+            final String token = jwtUtility.generateToken(userDetails);
+            return ResponseEntity.status(HttpStatus.OK).body(new JWTResponse(token, userFeign.getUserDetailsByEmail(loginRequest.getEmail())));
```

Comment:
try to break builder to next line so that it will be more readable i,e
ResponseEntity
.status(HttpStatus.OK)
.body()